tests/subsys/fs/fs_api/src/test_fs.c

```diff
@@ -73,7 +73,7 @@ static int temp_unlink(struct fs_mount_t *mountp, const char *path)
 }
 
 static int temp_rename(struct fs_mount_t *mountp, const char *from,
-			const char *to)
+		       const char *to)
```

Comment:
White space changes are not related to the issue being solved; they should have separate commit.

Reply:
Fixed

Reply:
If you want to follow through with the whitespace fixes you may another commit on top of this one or create new pull request.